assets/js/components/Work/Tabs/Administrative/Administrative.jsx

```diff
@@ -16,6 +16,7 @@ import WorkTabsAdministrativeCollection from "@js/components/Work/Tabs/Administr
 import useFacetLinkClick from "@js/hooks/useFacetLinkClick";
 import { formatDate } from "@js/services/helpers";
 import { IconEdit } from "@js/components/Icon";
+import useSearchTermClick from "@js/hooks/usePassedInSearchTeam";
```

Comment:
Should this be named `usePassedInSearchTerm` instead of `usePassedInSearchTeam`?